_rules/button-accessible-name-97a4e1.md

```diff
@@ -50,55 +50,55 @@ There are no major accessibility support issues known for this rule.
 
 #### Passed Example 1
 
-Regular button.
+The [accessible name][] of `button` elements comes from their content.
```

Comment:
If you want to be consistent with the other examples...

The accessible name of the button element comes from its content.